packages/lit-dev-content/site/blog/2023-09-27-lit-3.0-prerelease-2.md

```diff
@@ -0,0 +1,133 @@
+
+The full list of prerelease packages is:
+- `lit@3.0.0-pre.1`
+- `lit-html@3.0.0-pre.1`
+- `lit-element@4.0.0-pre.1`
+- `@lit/reactive-element@3.0.0-pre.1`
+- `@lit/context@1.0.0-pre.1`
+- `@lit/task@1.0.0-pre.1`
+- `@lit/react@1.0.0-pre.1`
+- `@lit/task@1.0.0-pre.1`
+- `@lit-labs/compiler@0.1.0-pre.1`
+- `@lit-labs/preact-signals@0.1.0-pre.1`
+
+## Lit 3.0 Prerelease 2
+
+As we [announced with the first prerelease](./2023-05-15-lit-3.0-prerelease/), Lit 3.0 is a breaking change that most notably removes support for IE11. Lit 3.0 is intended to have no new features, since new features can be added in non-breaking minor versions.
+
+### New hybrid decorators with standard decorators support
+
+One new thing we _have_ added to Lit 3.0 though is support for TC39 standard decorators. The new decorator spec has reached Stage 3 in TC39, meaning that browsers and compilers are now implementing them. This is a huge step for Lit because it allows us to begin the process of moving to decorator implementation that won't require a compiler to use.
+
+Recently TypeScript 5.2 added support for standard decorator metadata, which was the last piece needed for us to use the new specification (Babel's next release will have support). So we've made our decorators work with TypeScript's `experimentalDecorators` flag set to either `true` or `false`.
+
+The reason why we've included standard decorator support in a breaking change is that we've taken great care to make the upgrade path from experimental decorators smooth, and in order do that we've introduced a few minor breaking changes to our experimental decorator behavior to make the behavior consistent with the standard decorator behavior.
+
+The big thing is that we've added support for decorating the new "auto-accessors" - fields prefixed with the `accessor` keyword. Auto-accessors create a getter/setter pair and private storage that they access. These are able to be overridden by standard decorators.
+
+Standard decorators _require_ that fields use the `accessor` keyword. To facilitate incremental upgrades to standard decorators, we now supportm them in experimental decorators, so that the same callsite works with both settings:
+
+```ts
+class MyElement extends LitElement {
+  @property()
+  accessor myProperty = 'hello';
+}
+```
+
+This means that you can upgrade your decorated fields to use `accessor`, and once they all are, you can turn off `experimentalDecorators`.
+
+The breaking changes were required because it's not possible to differentiate between auto-accessors and hand-written accessors. Previously we wouldn't automatically call `this.requestUpdate()` for accessors, instead requiring the hand-written setter to call it manually. Now we do call `requestUpdate()` automatically. We also will read the value of an accessor on first render and use that as the initial value for `changedProperties` and attribute reflection.
```

Comment:
Since this is diving into some breaking changes, the change in `requestUpdate`, should other breaking changes be highlighted.

E.g. For custom property accessors (`@property`, or `@state`), decorators should now be set on the setter. This will be a breaking change for anyone who follows the recommendation on lit.dev.